### PR TITLE
Chipping away at Tailwind stuff

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# NOT READY
- (Edit the above to reflect status)

# Summary
- TL;DR - what's this PR for?

# Needed By (Date)
- When does this need to be merged by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Do this
1. Then this
2. Then this

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "tailwindcss": "^1.6.2"
   },
   "devDependencies": {
+    "@fullhuman/postcss-purgecss": "^2.3.0",
```
Comment (sherakama) [inline package.json:35]: I don't think we can use this here as we won't be using all of our generated css.

File: src/css/index.css [hunk 2]
```diff
@@ -1,15 +1,181 @@
...
+    @apply
+    inline-block
+    font-regular font-stanford
+    text-white;
+
+    line-height: 0.75;
+    -ms-font-feature-settings: "liga" 1;
+    font-feature-settings: "liga";
+    font-variant-ligatures: discretionary-ligatures;
+}
```
Comment (sherakama) [inline src/css/index.css:45]: Can these be moved into a component plugin?

Ideally, we would want everything to be in a plugin for maximum re-use and extensibility.

File: src/plugins/components/button/button.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+module.exports = function () {
```
Comment (sherakama) [inline src/plugins/components/button/button.js:1]: Niiiice.

File: src/plugins/components/button/button.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+      '.button': {
+        padding: '1rem 2rem 1.15rem',
+        display: 'inline-block',
+        width: 'auto',
+        border: 'none',
+        backgroundColor: theme('colors.digital-red.default'),
+        color: theme('colors.white'),
+        textAlign: 'center',
+        fontFamily: sans,
+        fontWeight: '400',
```
Comment (sherakama) [inline src/plugins/components/button/button.js:15]:
```suggestion
        fontWeight: theme('fontWeight.normal'),
```

File: package.json (hunk 1 above)
Comment (sherakama) [reply package.json:35]: We can use this in our projects that use this version of decanter but ootb we will need to have everything outputted.

File: src/css/index.css (hunk 2 above)
Comment (yvonnetangsu) [reply src/css/index.css:45]: Agree - will do :)

File: package.json (hunk 1 above)
Comment (yvonnetangsu) [reply package.json:35]: Yup - agree. Just testing. I've removed the purge stuff and saving the code for production use 😺

**Pull request merged**